tests/shell/shell_recorder/tutorial_wrapper/CMakeLists.txt

```diff
@@ -34,6 +34,9 @@ add_msr_test (tutorial_storage_plugins "${CMAKE_SOURCE_DIR}/doc/tutorials/storag
 add_msr_test (cmerge "${CMAKE_SOURCE_DIR}/doc/tutorials/cmerge.md" REQUIRED_PLUGINS hosts line PROPERTY LABELS memleak)
 add_msr_test (tutorial_namespaces "${CMAKE_SOURCE_DIR}/doc/tutorials/namespaces.md")
 
+# This is test is very long running and very resource intensive. Please run this run validation for this tutorial manually
```

Comment:
### Review

I think there is a redundant 'is':
```suggestion
# This test is very long running and very resource intensive. Please run this run validation for this tutorial manually
```

**Documentation** 
The documentation is clear and understandable. I think the removal of the redundant docker/podman steps is the right approach for refactoring this tutorial.

**Examples**
The examples are well chosen and understandable.

**Code is conforming to our Coding Guidelines**
Yes.

**APIs are conforming to our Design Guidelines**
Yes.

**Code is consistent to our Design Decisions**
Yes.